functions.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require_once 'dbConnect.php';
+
+/**
+ * function to call a var dump with pre tags
+ * @param $variabletoDump - the variable to be dumped
+ */
+function var_dumpPre($variabletoDump) {
+    echo '<pre>';
+    var_dump($variabletoDump);
+    echo '</pre>';
+}
+
+/**
+ * get the collection data from the db
+ * @param $db - the database connection
```

Comment:
line breaks separating desc, param and return
(ill just comment this on this one - please repeat for all)